packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js

```diff
@@ -5,7 +5,7 @@ const webpack = require('webpack')
 const readline = require('readline')
 const { Worker } = require('worker_threads')
 
-const workerThreads = 8
+const workerThreads = 10
```

Comment:
Upping this for now since if we're paying for a beefier machine, we might as well leverage it.